src/design-tokens/tier-2-usage/typography-usage.css

```diff
@@ -90,27 +90,31 @@
 
 /* Old / legacy heading sizes */
```

Comment:
Here's the mixin updates (adding this comment so I can link to it somewhere else).